### DISCUSSION TITLE
Add "tags" support for MDX <Meta />

### DISCUSSION THREAD

**Discussion by kroeder** [storybookjs/storybook#26348] [Ideas]
### Is your feature request related to a problem? Please describe.

As discussed today with @valentinpalkovic and @shilman, there is no way to add tags within MDX `<Meta tags={["deprecated"]} />`

In story-files, the tags can be added and re-used in the `renderLabel` custom function. 
With docs-mode, this is currently not possible. 

### Describe the solution you'd like

Add `<Meta tags={} />` input to the `Meta` component in MDX.

### Are you able to assist to bring the feature to reality?

yes, I can

**Comment by YossiSaadi** [comment]
Please add it :)
Willing to try and contribute as well if there's something I can help with!

**Reply by shilman** [reply]
This should be fixed in the latest release. Please lmk if you're running into problems with it.

```
npx storybook@latest upgrade
```
